src/BasicLinkAsserter.js

```diff
@@ -13,10 +13,15 @@ class BasicLinkAsserter {
     let linksSet = new Set(linkify.find(botMsg.messageText)
       .map(u => u.href))
     if (botMsg.buttons) {
-      botMsg.buttons.forEach(b => linksSet.add(b.imageUri))
+      botMsg.buttons.forEach(b => {
+        linksSet.add(b.payload)
```

Comment:
check if payload set first. or filter the linksSet below to not contain any null values (otherwise, hasSubStringOfEntry will fail). same applies for line 23